pkg/plugin/watch.go

```diff
@@ -169,6 +170,15 @@ func (e *AutoscaleEnforcer) watchVMEvents(
 					submitVMDisabledScaling(name)
 				}
 
+				if !oldInfo.ScalingEnabled && !newInfo.ScalingEnabled && oldInfo.Using() != newInfo.Using() {
```

Comment:
I think I was mistaken when I suggested this condition. It should actually be:

```go
if (!oldInfo.ScalingEnabled || !newInfo.ScalingEnabled) && oldInfo.Using() != newInfo.Using() {
    ...
}
```

Because otherwise we won't handle usage changes if happens at the same time as scaling being enabled/disabled.

Reply:
Good catch.